api/server/router/container/backend.go

```diff
@@ -18,6 +18,7 @@ type execBackend interface {
 	ContainerExecInspect(id string) (*backend.ExecInspect, error)
 	ContainerExecResize(name string, height, width int) error
 	ContainerExecStart(ctx context.Context, name string, stdin io.Reader, stdout io.Writer, stderr io.Writer) error
+	ContainerExecKill(ctx context.Context, name string, signame string) error
```

Comment:
Wondering if we should make this take a (to be created) `types.ExecKillConfig`, so that more options can be added without making a long list of arguments (I'll leave another comment with some thoughts why this _could_ be needed; also see https://github.com/moby/moby/pull/43206).